test/Conversion/amd/decompose-unsupported-conversions.mlir

```diff
@@ -1,8 +1,9 @@
-// RUN: triton-opt %s --split-input-file --decompose-unsupported-amd-conversions=arch=gfx942 | FileCheck %s
```

Comment:
I wanted to rename this file to test/Conversion/amd/decompose-unsupported-conversions**-gfx11**.mlir, but git marks it as new file and looses history. So I've left it as-is for now.